students/hattie2/Session11/danceability.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python3
+
+import pandas as pd
+music = pd.read_csv("featuresdf.csv")
+
+stuff = music.loc[music['danceability'] > 0.8]
+
+sorted_stuff = stuff.sort_values(['danceability'], ascending=False)
+
+quiet_stuff = sorted_stuff.loc[music['loudness'] < -5]
```

Comment:
this might be using Pandas to power your way through, but can you try it with a list comprehension since that's the point of this assignment?